numpy/meson.build

```diff
@@ -369,7 +369,7 @@ install_subdir('tests', install_dir: np_dir, install_tag: 'tests')
 compilers = {
   'C': cc,
   'CPP': cpp,
-  'CYTHON': meson.get_compiler('cython')
+  'CYTHON': cy,
```

Comment:
Clearly right, but I won't pretend to understand how these are all defined here.

Reply:
`meson.build` files inherit the namespace of `meson.build` files defined in parent folders. That means that the `cc`, `cpp`, and `cy` references to the compiler objects defined in the root `meson.build` are available in any subfolder.